binaries/speed_benchmark_torch.cc

```diff
@@ -83,10 +85,15 @@ int main(int argc, char** argv) {
 
   std::vector<std::string> input_dims_list = split(';', FLAGS_input_dims);
   std::vector<std::string> input_type_list = split(';', FLAGS_input_type);
+  std::vector<std::string> input_format_list = split(';', FLAGS_input_format);
```

Comment:
Would it make sense to call this arg `input_memory_format` instead of `input_format`? I know `input_memory_format` is mouthful but I thought it is more explicit. Just a suggestion.

Reply:
Thanks, will change.

Reply:
Done.